patches/goyaml-pr-241.diff

```diff
@@ -1,31 +0,0 @@
-This is the diff from PR #241 on goyaml.
-When this file is removed, also remove the Skip from the test
-featuretests/cmdjuju_test.go TestServiceGetWeirdYAML.
```

Comment:
We should remember to do this when we have _yaml_ at the required hash.

Reply:
@manadart good shout, totally missed this.